src/main/java/net/masterthought/cucumber/generators/TagsOverviewPage.java

```diff
@@ -1,19 +1,18 @@
 package net.masterthought.cucumber.generators;
 
+import net.masterthought.cucumber.Configuration;
+import net.masterthought.cucumber.ReportResult;
+import net.masterthought.cucumber.json.support.Status;
+import net.masterthought.cucumber.json.support.TagObject;
+import org.apache.commons.lang3.StringUtils;
+
```

Comment:
It's a result of IntelliJ's automatic import organization. By default, it has the following order:
1. All non java.* and javax.* packages, sorted
2. java.* and javax.* packages, sorted
3. Static imports, sorted